integration-tests/cli/csv-parser-price.it.js

```diff
@@ -119,6 +116,26 @@ describe('CSV and CLI Tests', () => {
         },
       )
     })
+
+    test('should log errors to stderr and log file', (done) => {
```

Comment:
The issue was, that when we used the stdout for data output, we were printing there also other log messages so the resulting JSON was polluted and it was not possible to parse it.

Now we use stdout only for content (all log messages go to log file) and if there is any error, we stop processing and log it to stderr. 

So no more info/warn/debug/.. logs in stdout unless we output data to a file. Also as you said, we do not continueOnProblems so we can print an error message to stderr and quit processing because the output will be an invalid JSON due to an error anyway.

So the main difference is that we do not print non-error messages to stderr.